.github/workflows/old-ghcs.skip.yml

```diff
@@ -0,0 +1,39 @@
+name: Validate old ghcs Skip
+
+# This Workflow is special and contains a workaround for a known limitation of GitHub CI.
```

Comment:
Instead of going in all the details, I'd reference the f other skip workflow file

Reply:
I just duplicated what both the existing skip workflows do.